### PR TITLE
Feature/add redux and axios

### PR DESCRIPTION
**Pull request by sophiasagan**

### CODE REVIEW

**Review by hmerritt — CHANGES_REQUESTED** [review]
Everything looks great with Redux + Axios auth!

One small suggestion: `.env.sample` was deleted when making `.env` but it seems like `.env.sample` is needed for the tests to pass.

It would be great if you could restore `.env.sample` (even though we don't need it ourselves, just for the tests to pass)

**Review by hmerritt — APPROVED** [review]
Looks good! (tests passed which is great 👍 )

**Comment by josegocampo** [comment]
wow Sophia, you are amazing! good job on this, also good catch Harry!

File: src/state/actions/userActions.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { INITIALIZE_USER } from './userActionTypes';
+
+export const initializeUser = (authService, history) => async dispatch => {
+  try {
+    var user = await authService.getUser();
```
Comment (josegocampo) [inline src/state/actions/userActions.js:5]: is there a reason you are using var here :O  just wondering

**Review by josegocampo — APPROVED** [review]
Looks great, thanks a lot for doing this Sophia!

**Pull request merged**